qutebrowser/browser/webkit/webkittab.py

```diff
@@ -693,6 +764,13 @@ def event_target(self):
     def shutdown(self):
         self._widget.shutdown()
 
+    def run_js_sync(self, code, *, world=None):
```

Comment:
The reason pylint wants a docstring here is because:

- The method is public
- The method is longer than 3 lines

Might be good opportunity to point out that this is QtWebKit-specific here as well.